### PR TITLE
Done: added main homework lesson 03

### PR DESCRIPTION
**Pull request by max3koz**
Done: added main homework lesson 03

### CODE REVIEW

File: lesson_03/homework_03.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті
+alice_in_wonderland = '"Would you tell me, please, which way I ought to go from here?"\n'\
```
Comment (DartBrovsky) [inline lesson_03/homework_03.py:3]: You have tried using \n symbol find all ' as I understand. But for task 02 can only do like ->
`for symbol in alice_in_wonderland:
      if symbol == "'": 
          print(symbol)`
Or sth like that but in this way you are able to find and show all ' symbols.

File: lesson_03/homework_03.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+для даного її замовлення.
+Назва товару    Кількість   Ціна
+Піца велика     4           274 грн
+Піца середня    2           218 грн
+Сік             4           35 грн
+Торт            1           350 грн
+Вода            3           21 грн
+"""
+
+print("\nTask08. \n"
```
Comment (DartBrovsky) [inline lesson_03/homework_03.py:108]: Interesting variant of solution)

**Review by DartBrovsky — APPROVED** [review]
I approve your PR but you can take a look to my comment regarding 2nd task and try to solve it in another way. Thanks you

File: lesson_03/homework_03.py (hunk 1 above)
Comment (max3koz) [reply lesson_03/homework_03.py:3]: Updated task_02 in the commit cf9e13f84c6c588b5f0cd2707e4f3e51c347085c

File: lesson_03/homework_03.py (hunk 2 above)
Comment (max3koz) [reply lesson_03/homework_03.py:108]: Thanks for comment!

**Pull request merged**